src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -57,9 +57,10 @@ contains the following details:
 | `--kubecontext` `<string>`            | kubeconfig context to use
 | `--repository` `<string>`             | The repository from where the various Submariner images will be sourced (default "quay.io/submariner")
 | `--version` `<string>`                | Image version
-| `--service-discovery`                 | Enable Multi Cluster Service Discovery (default true)
+| `--components <strings>`              | The components to be installed - any of service-discovery,connectivity,globalnet. The defaults are: `service-discovery`,`connectivity`
 | `--globalnet`                         | Enable support for overlapping Cluster/Service CIDRs in connecting clusters (default disabled)
```

Comment:
On a side note, should `globalnet`  be deprecated and removed from the docs as well?

Reply:
good point, I see it still part of the listed flags in the deploy-broker command, so need to remove it from there as will

Reply:
yes - it would be good to clean it up in the code.